pre-cache/olm

```diff
@@ -17,13 +13,9 @@ extract_pull_spec(){
 
 render_index(){
     index=$1
-    pull_secret_path=$2
```

Comment:
I understand that pull_secret_path is no longer needed because it would use the default or something. Overall, would you please put some description on the PR/commit message? I think it'd be good for the future us. Thanks!

Reply:
Done